google/cloud/tasks/v2beta3/cloudtasks.proto

```diff
@@ -511,7 +559,10 @@ message CreateTaskRequest {
   // `projects/PROJECT_ID/locations/LOCATION_ID/queues/QUEUE_ID`
   //
   // The queue must already exist.
-  string parent = 1;
+  string parent = 1 [
```

Comment:
I think `CreateTaskRequest.task` should also be required, according to the GAPIC config.